### ISSUE TITLE
OSSEC-DBD error on invalid element in rules file. 

### ISSUE THREAD

**Issue by jgreene-VSLLC** [wazuh/wazuh-ruleset#107]
**Versions:**
Wazuh 3.3.0 (bug was present in previous versions)
ossec-dbd enable = true

**Problem:**

ossec-db will not start properly and presents errors regarding invalid value in an element. 

**ossec.jason errors:**

```
{"timestamp":"2018/03/13 12:11:11","tag":"ossec-dbd","level":"error","description":"(1235): Invalid value for element 'options': no_full_log."}
{"timestamp":"2018/03/13 12:11:11","tag":"ossec-dbd","level":"error","description":"(1238): Invalid value for element 'options': no_full_log"}
{"timestamp":"2018/03/13 12:11:11","tag":"ossec-dbd","level":"error","description":"(1220): Error loading the rules: 'ruleset/rules/0520-vulnerability-detector.xml'."}
{"timestamp":"2018/03/13 12:11:11","tag":"ossec-dbd","level":"critical","description":"(1202): Configuration error at '/var/ossec/etc/ossec.conf'."}
{"timestamp":"2018/03/13 12:11:19","tag":"ossec-remoted","level":"error","description":"(1501): No IP or network allowed in the access list for syslog. No reason for running it. Exiting."}
{"timestamp":"2018/03/13 12:11:27","tag":"ossec-dbd","level":"error","description":"(1235): Invalid value for element 'options': no_full_log."}
{"timestamp":"2018/03/13 12:11:27","tag":"ossec-dbd","level":"error","description":"(1238): Invalid value for element 'options': no_full_log"}
{"timestamp":"2018/03/13 12:11:27","tag":"ossec-dbd","level":"error","description":"(1220): Error loading the rules: 'ruleset/rules/0520-vulnerability-detector.xml'."}
{"timestamp":"2018/03/13 12:11:27","tag":"ossec-dbd","level":"critical","description":"(1202): Configuration error at '/var/ossec/etc/ossec.conf'."}
{"timestamp":"2018/03/13 12:13:48","tag":"ossec-remoted","level":"error","description":"(1501): No IP or network allowed in the access list for syslog. No reason for running it. Exiting."}
{"timestamp":"2018/03/13 12:13:58","tag":"ossec-dbd","level":"error","description":"(1235): Invalid value for element 'options': no_full_log."}
{"timestamp":"2018/03/13 12:13:58","tag":"ossec-dbd","level":"error","description":"(1238): Invalid value for element 'options': no_full_log"}
{"timestamp":"2018/03/13 12:13:58","tag":"ossec-dbd","level":"error","description":"(1220): Error loading the rules: 'ruleset/rules/0520-vulnerability-detector.xml'."}
{"timestamp":"2018/03/13 12:13:58","tag":"ossec-dbd","level":"critical","description":"(1202): Configuration error at '/var/ossec/etc/ossec.conf'."}
{"timestamp":"2018/03/13 12:14:44","tag":"ossec-dbd","level":"error","description":"(1235): Invalid value for element 'options': no_full_log."}
{"timestamp":"2018/03/13 12:14:44","tag":"ossec-dbd","level":"error","description":"(1238): Invalid value for element 'options': no_full_log"}
{"timestamp":"2018/03/13 12:14:44","tag":"ossec-dbd","level":"error","description":"(1220): Error loading the rules: 'ruleset/rules/0520-vulnerability-detector.xml'."}
{"timestamp":"2018/03/13 12:14:44","tag":"ossec-dbd","level":"critical","description":"(1202): Configuration error at '/var/ossec/etc/ossec.conf'."}
```

**Referencing the following file:**
/var/ossec/ruleset/rules/0520-vulnerability-detector.xml

Note: Commenting out the 'no_full_log' element allows ossec-dbd to start normally. 

`<!--    <options>no_full_log</options>  -->`

**Comment by vikman90** [comment]
Hi @jgreene-VSLLC,

This error was caused by a bug in the rule parser for `ossec-dbd` (that is different from the one for `ossec-analysisd`).

In fact I found two options that weren't parsed: 
- `no_full_log`
- `no_counter`

This commit (branch 3.2) should solve it: https://github.com/wazuh/wazuh/commit/fccb99fcbe0d3efc8148da673a910cf34bba956f

I've just merged it into master (future 3.3).

Thank you very much for reporting this bug!
Best regards.
